src/main/java/seedu/duke/data/timetable/RecurringEvent.java

```diff
@@ -1,23 +1,28 @@
 package seedu.duke.data.timetable;
 
+import seedu.duke.ui.InterfaceManager;
+
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.util.ArrayList;
 
 public abstract class RecurringEvent extends Event {
-    private LocalDate endRecurrence;
-    private String recurrenceType;
+    private final LocalDate endRecurrence;
+    private final String recurrenceType;
```

Comment:
If final I think it should be in caps
```suggestion
    private final LocalDate END_RECURRENCE;
    private final String RECURRENCE_TYPE;
```

Reply:
Apologies, forgot to refactor the names after changing it to final.